File: selenide/SelenideTAF/src/test/java/examplesScripts/plainSelenium.java
package examplesScripts;

import org.openqa.selenium.*;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.junit.jupiter.api.Test;

public class plainSelenium {
	
	@Test
	public void launch_demosite() {
		WebDriver driver = new FirefoxDriver();
		driver.get("http://5elementslearning.dev/demosite");
		driver.findElement(By.linkText("My Account")).click();
		if(driver.getPageSource().contains("Welcome, Please Sign In")) {
			driver.quit();
		}else {
			driver.quit();
		}
	}
}

File: seleniumbase/SeleniumBaseExample/ExamplesScripts/plainSelenium.py
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Firefox()
driver.get("https://5elementslearning.dev/demosite/")
driver.implicitly_wait(0.5)
driver.find_element(by=By.LINK_TEXT, value="My Account").click()
message = driver.page_source
temp="Welcome, Please Sign In"
if(temp in message):
	driver.quit()
	
	